### PR TITLE
Add command line setup command

### PR DESCRIPTION
**Pull request by sypets**
The setup command is quite new. Also, we may need to remind people they are not in a composer installation and where the command is located.

### CODE REVIEW

File: Documentation/Appendix/SettingUpTypo3Manually.rst [hunk 1]
```diff
@@ -54,9 +54,26 @@ Create FIRST_INSTALL
...
-Continue ...
-============
+Installation wizard
+===================
 
 You can now access the TYPO3 installation wizard by going to, e.g.
 http://t3coredev/typo3/. Follow the instructions of the installation
 wizard.
+
+Or run typo3 setup on the command line. Note that you are effectively in a
```
Comment (linawolf) [inline Documentation/Appendix/SettingUpTypo3Manually.rst:64]:
```suggestion
Or run `typo3 setup` on the command line. You are in a
```

**Review by linawolf — COMMENTED** [review]
Thanks a lot for these changes, just a few small remarks

**Comment by sypets** [comment]
Resolved review comment and rebased with fixing conflicts.

File: Documentation/Appendix/SettingUpTypo3Manually.rst [hunk 2]
```diff
@@ -54,11 +54,28 @@ Create FIRST_INSTALL
...
+===================
 
 You can now access the TYPO3 installation wizard by loading the page in the
 browser with the configured URL, e.g. http://t3coredev. This should redirect
 to http://t3coredev/typo3/install.php.
 
 Follow the instructions of the installation wizard.
+
+Or run `typo3 setup` on the command line. You are in a
+non-composer installation and the command is in
```
Comment (brotkrueml) [inline Documentation/Appendix/SettingUpTypo3Manually.rst:67]:
```suggestion
non-Composer installation and the command is in
```

**Review by brotkrueml — APPROVED** [review]

**Review by linawolf — APPROVED** [review]

**Pull request merged**